mkdocs.yml

```diff
@@ -24,6 +24,7 @@ nav:
     - Home: llb/index.md
     - Feature API Specification: llb/llb-api-spec.md
     - Auth API Specification: llb/llb-auth-api.md
+    - Testing Specification: llb/llb-test-spec.md
```

Comment:
Isn't this something which applies to all projects? Could we move it to somewhere general? If so copy the structure from #14 `mkdocs.yml` to prevent merge conflicts